examples/advanced/inhibitory_gating.ipynb

```diff
@@ -8,10 +8,11 @@
   {
    "cells": [
     {
-     "cell_type": "markdown",
+     "cell_type": "heading",
+     "level": 1,
      "metadata": {},
      "source": [
-      "# Nengo Example: Inhibitory Gating of Ensembles"
+      "Nengo Example: Inhibitory Gating of Ensembles"
```

Comment:
Is there a reason for changing the markdown headings to cells of the the type "heading"? Current Jupyter version do not use the "heading" cell type anymore. While these cells will automatically converted, I would prefer to keep the markdown cells.